internals/cli/cmd_enter_test.go

```diff
@@ -172,6 +175,7 @@ func (s *PebbleSuite) TestEnterExecListDir(c *C) {
 }
 
 func (s *PebbleSuite) TestEnterExecReadServiceOutputFile(c *C) {
+	plan.UnregisterSectionExtension(workloads.WorkloadsField)
```

Comment:
It is not clear to me when this is needed, or why this is needed. It is not consistently needed, which makes me think I do not understand. Can this not be, for example, moved to the test teardown with an explanation why this is needed ?